mail_alias_manager/db/models/sending_alias.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy.sql.schema import Column
+from sqlalchemy_utils.types.email import EmailType
+
+from ..db import DB, MODEL
+
+
+class SendingAlias(MODEL):
```

Comment:
Maybe call this `SenderAlias` or `SenderIdentity`. If you feel like `SenderIdentity` is better, then also rename the `alias` attribute to `identity`.

Reply:
I think SendingAlias is better